.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Fixes #  
+
+### Changes proposed in this pull request: 
+
+ - 
+ - 
```

Comment:
The checklist can include:
- [ ] Documentation (if applicable)
- [ ] Test code (Unit Tests are highly preferred)
- [ ] Link to Issue `e.g. Resolves #NNN`